src/rust/wpcomics/sources/nettruyen/res/source.json

```
@@ -4,7 +4,7 @@
 		"lang": "vi",
 		"name": "NetTruyen",
 		"version": 10,
```

version should be incremented
```suggestion
		"version": 11,
```